config/custom_items.txt

```diff
@@ -539,6 +539,12 @@ character_name: Fortune Bloise
 item_path: /obj/item/weapon/storage/backpack/satchel/fluff/swat43bag
 }
 
+{
+ckey: SpoopyLizz
```

Comment:
These are in ckey format, which needs to be all lowercase, no spaces etc, so this should probably be `spoopylizz`